content/zones/derelict-outpost.json

```diff
@@ -0,0 +1,47 @@
+loadout: basicFoundation
+startingItems: [
+  copper/1000
+  lead/300
+  metaglass/200
+  graphite/200
+  titanium/100
+  silicon/200
+  iron/100
+  steel/100
+]
+configureObjective: {
+  type: Launched
+  zone: derelict-outpost
+}
+requirements: [
+  {
+    type: Launched
+    zone: front-gate
+  }
+  {
+    type: ZoneWave
+    zone: nuclearComplex
+    wave: 45  
+  }
+  {
+    type: Unlock
+    block: phantasm-factory
+  }
+  {
+    type: Unlock
+    block: fortress-factory
+  }
+]
+resources: [
+  copper
+  lead
+  coal
+  titanium
+  iron
+  thorium
+  uranium
+  sand
+  scrap
+  cobblestone
```

Comment:
Just changed "cobblestone" to "gravel" - this may need to be changed, depending on if my pull request is accepted

Reply:
I'll change it later, I gtg to the police station to make my drivers license